static/src/javascripts/components/curl/curl-domReady.js

```diff
@@ -1,4 +1,4 @@
-(function(){/*
+window.require || (function(){/*
```

Comment:
This is a hack! It seems that loading `curl` twice makes things fail, so this just stops the `app.js` copy of curl from loading if there is already one in the page.